Hyper/Window.cs

```diff
@@ -10,27 +9,30 @@ public class Window : GameWindow
 {
     private Game _game;
 
-    private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
+    private CommandInterpreter _interpreter;
 
     public Window(GameWindowSettings gameWindowSettings, NativeWindowSettings nativeWindowSettings)
         : base(gameWindowSettings, nativeWindowSettings)
```

Comment:
How about some way to change the starting geometry in Program.cs to spherical? Right not debugging spherical is annoying because you have to launch euclidean and than use commands to get spherical which takes a lot of time

Reply:
I'll change it so that the menu pops out first in release and in debug geometry info will be passed through command line argument